src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,86 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import { SelectedSex } from '../types/SelectedSex';
+
+type PeopleFiltersProps = {
+  query: string,
+  sex: string,
+  centuries: string[],
+  handleSex: (newSex: string) => void,
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+};
+
+export const PeopleFilters: React.FC<PeopleFiltersProps> = ({
+  query,
+  sex,
+  centuries,
+  handleSex,
+  handleQueryChange,
+}) => {
+  const allCentury = ['16', '17', '18', '19', '20'];
+
+  function toggleCentury(century: string) {
+    return centuries.includes(century)
+      ? centuries.filter(num => num !== century)
+      : [...centuries, century];
+  }
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.

Reply:
Not fixed 😢